steps/wsclean.cwl

```diff
@@ -50,27 +50,29 @@ arguments:
  - $(inputs.size_x)
  - $(inputs.size_y)
  - prefix: -temp-dir
-   valueFrom: /tmp
```

Comment:
this is broken on OSX https://github.com/common-workflow-language/cwltool/issues/682

Reply:
Bummer. I'll back it out then.

Reply:
Fixed